cg/constants/archiving.py

```diff
@@ -1,6 +1,6 @@
 from enum import StrEnum
 
-DEFAULT_SPRING_ARCHIVE_COUNT = 100
+DEFAULT_SPRING_ARCHIVE_COUNT = 500
```

Comment:
what is the purpose of this change?

Reply:
This looks like an artefact of branching from master instead of the feature branch. Isak just merged a PR changing it from 100 to 500 in master

Reply:
Yes, indeed.